jep/200/README.adoc

```diff
@@ -276,7 +292,86 @@ For similar reasons, certain tests written in Groovy rather than Java prevent no
+
+* Remoting
+* link:https://github.com/jenkinsci/docker-slave[docker-slave] (including the `latest` tag)
+* link:https://github.com/jenkinsci/docker-jnlp-slave[docker-jnlp-slave] images (including the `latest` tag)
+* link:https://github.com/jenkinsci/swarm-plugin/[Swarm Plugin Client]
+
+The Remoting release **may** be integrated into the weekly release before link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120]
+gets integrated.
+In such case new API will not be used, and hence there will be no behavior change.
+
+=== Phase 2. Weekly Release
+
+Target date: Jan 06, 2017
+
+0. Once this JEP is approved, the https://jenkins.io/redirect/class-filter/ will be created on the Jenkins website
+** This document should provide a custom guide for creating JIRA issues with the `jep-200` label
+0. The JEP sponsor will write an announcement blogpost, which will describe the change and provide links to mitigation guidelines
+** This blog-post will be used as part of the upgrade guideline for LTS
+** Upgrade guidelines should explicitly recommend backing up the instance before the upgrade
+** Upgrade guidelines will also provide whitelisting guidelines to plugin developers
+0. Once the blog post draft is approved,
+link:https://github.com/jenkinsci/jenkins/pull/3120[Jenkins PR #3120] will be integrated towards the next weekly release
+
+After the weekly release the JEP sponsor
+(or a group of people nominated by him, _JEP-200 maintainer(s)_)
+will be responsible to provide an extra support for the issues:
+
+* _JEP-200 maintainer(s)_ will regularly review open defects and triage them
+* _JEP-200 maintainer(s)_ may request additional information from the reporter.
+Finally, they are expected to communicate the triage outcome.
+* Possible triage outcomes:
+** Accepted - patch in the plugin. to be done by _JEP-200 maintainer(s)_
+** Accepted - update whitelist in the core (similar to link:https://github.com/jenkinsci/jenkins/pull/3120/files#diff-ff24cb984ddd641f49a22cc13a90cfd3[these cases]), to be done _JEP-200 maintainer(s)_
+** Rejected - functional defect.
+_JEP-200 maintainer(s)_ are **not responsible** to fix any issue,
+the reporter can use the suggested workarounds.
+The issue remains open as a common bug.
+** Rejected - security risk.
+In such case the issue will be moved to the SECURITY bugtracker and then handled by the link:https://jenkins.io/security/[Jenkins Security team]
+* For accepted issues _JEP-200 maintainer(s)_ schedule the fix and communicate ETAs to the reporter
+
+=== Phase 3. LTS availability
+
+Target Date: Mar 14, 2018
+
+There is no plan to backport the proposed change to the 2.89.x LTS baseline.
+The change will be integrated into the LTS if the
+link:https://wiki.jenkins.io/display/JENKINS/Governance+Meeting+Agenda[governance meeting]
+selects a weekly with the integrated change.
+The weekly may be rejected if the release is not stable enough.
```

Comment:
All this stuff is potentially applicable to any JEP proposing to touch core. It does not need to be discussed here; it is just follows from normal Jenkins procedures.